qemu/tests/cfg/boot_nic_with_intel_iommu.cfg

```diff
@@ -0,0 +1,18 @@
+- boot_nic_with_intel_iommu:
+    only q35
+    only virtio_net
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    type = boot_nic_with_intel_iommu
+    extra_params = "-device intel-iommu,intremap=on,device-iotlb=on"
+    machine_type_extra_params = "kernel-irqchip=split"
+    virtio_dev_disable_legacy = on
+    virtio_dev_disable_modern = off
+    virtio_dev_iommu_platform = on
+    virtio_dev_ats = on
+    vhostforce = on
+    pci_controllers += "pcie_nic"
+    type_pcie_nic = "ioh3420"
+    pci_bus_nic1 = pcie_nic
+    Linux:
```

Comment:
is this `Linux:` necessary?

Reply:
I think so. No need to set this value for Windows. Besides, the param is added to the linux guest kernel cmd line. 
Anyway, if we remove "Linux:", this case also works fine.